F1WM/Controllers/NewsController.cs

```diff
@@ -69,7 +105,13 @@ public async Task<IEnumerable<NewsTagCategory>> GetTagCategories()
 		[HttpGet("important")]
 		public async Task<IEnumerable<ImportantNewsSummary>> GetImportantNews()
 		{
-			return await service.GetImportantNews();
+			var responseData = cachingService.TryGetCacheValue<IEnumerable<ImportantNewsSummary>>(CacheKeys.ImportantNews.ToString());
+			if (responseData is null || responseData.Any())
+			{
+				responseData = await service.GetImportantNews();
+				cachingService.Set(CacheKeys.ImportantNews.ToString(), responseData, TimeSpan.FromDays(5));
```

Comment:
Can we afford to make cache duration so long? News change quite frequently.

Reply:
For now we will clear cache on every edit in db, so it's safe. I know that this solution is not perfect, but our old CMS is quite not friendly as for adding new features :)